robots/hsrb4s.urdf.xacro

```diff
@@ -45,11 +45,15 @@ POSSIBILITY OF SUCH DAMAGE.
     <xacro:include filename="$(find hsr_description)/urdf/wrist_v0/wrist.urdf.xacro" />
     <xacro:include filename="$(find hsr_description)/urdf/hand_v0/hand.urdf.xacro" />
 
+    <!-- This switches a format of mesh files for visual property -->
+    <xacro:arg name="use_obj_for_visual" default="false" />
+    <xacro:property name="g_use_obj_for_visual" value="$(arg use_obj_for_visual)"/>
```

Comment:
What does this prefix "g_" mean?

Reply:
`g_` means the property is defined in global scope.

Reply:
I got it.